### PR TITLE
add a "cross-eval" command to launch_eval, making it easier to launch…

### PR DESCRIPTION
**Pull request by amj**
… eval matches with new models

### CODE REVIEW

File: cluster/evaluator/launch_eval.py [hunk 1]
```diff
@@ -28,6 +28,43 @@
 import time
 
 
+def cross_eval(m1_path=None, m2_path=None, jobname=None, completions=20):
+    """
+    black_path, white_path: full gs:// paths to the .pb files to match up
```
Comment (sethtroisi) [inline cluster/evaluator/launch_eval.py]: m1_path, m2_path

File: cluster/evaluator/launch_eval.py [hunk 2]
```diff
@@ -28,6 +28,43 @@
...
+    kubernetes.config.load_kube_config()
+    configuration = kubernetes.client.Configuration()
+    api_instance = kubernetes.client.BatchV1Api(
+        kubernetes.client.ApiClient(configuration))
+
+    raw_job_conf = open("cluster/evaluator/cc-evaluator.yaml").read()
+
+    os.environ['MODEL_BLACK'] = m1_path
+    os.environ['MODEL_WHITE'] = m2_path
+    os.environ['JOBNAME'] = jobname + 'bw'
```
Comment (sethtroisi) [inline cluster/evaluator/launch_eval.py]: _bw

File: cluster/evaluator/launch_eval.py [hunk 3]
```diff
@@ -28,6 +28,43 @@
...
+    env_job_conf = os.path.expandvars(raw_job_conf)
+
+    job_conf = yaml.load(env_job_conf)
+    job_conf['spec']['completions'] = completions
+
+    resp = api_instance.create_namespaced_job('default', body=job_conf)
+
+    os.environ['MODEL_WHITE'] = m1_path
+    os.environ['MODEL_BLACK'] = m2_path
+    os.environ['JOBNAME'] = jobname + 'wb'
```
Comment (sethtroisi) [inline cluster/evaluator/launch_eval.py]: _wb

File: cluster/evaluator/launch_eval.py [hunk 4]
```diff
@@ -28,6 +28,43 @@
...
+    """
+    if m1_path is None or m2_path is None or jobname is None:
+        print("call me better")
+        return
+    kubernetes.config.load_kube_config()
+    configuration = kubernetes.client.Configuration()
+    api_instance = kubernetes.client.BatchV1Api(
+        kubernetes.client.ApiClient(configuration))
+
+    raw_job_conf = open("cluster/evaluator/cc-evaluator.yaml").read()
```
Comment (sethtroisi) [inline cluster/evaluator/launch_eval.py]: This file doesn't include a JOBNAME did you forget to git add it?

Comment (amj) [reply cluster/evaluator/launch_eval.py]: err, yes it's right above?

File: cluster/evaluator/launch_eval.py (hunk 2 above)
Comment (amj) [reply cluster/evaluator/launch_eval.py]: Done.

File: cluster/evaluator/launch_eval.py (hunk 3 above)
Comment (amj) [reply cluster/evaluator/launch_eval.py]: Done.  (actually, k8s can't have underscores in the name so its a hyphen but close enough ;)

**Pull request merged**